mlos_bench/mlos_bench/config/schemas/tunables/tunable-params-schema.json

```
@@ -73,6 +73,12 @@
                     "minItems": 2,
                     "maxItems": 2
                 },
+                "quantization": {
+                    "type": "number"
```

```suggestion
                    "type": "number",
                    "minimum": 1,
```

Also, is it a "number", or an "integer"?